### DISCUSSION TITLE
require is not defined in basic webhook

### DISCUSSION THREAD

**Discussion by ralphilius** [AntlerVC/firetable#1321] [Support / Q&A] [answered]
I'm trying to use the require as in the demo but encountered the error:

```ReferenceError: require is not defined```

```const lodash = require('lodash');```

**Comment by BeeBombshell** [comment]
Hey @ralphilius, `require is not defined` is a runtime error. Not sure what's causing it. Perhaps could you share your Webhook parser so that I can test it out?

**Reply by ralphilius** [reply]
@BeeBombshell Please see below for an example. I need to wrap the whole body in try/catch so that I could see the error

```javascript
const basicParser: Parser = async({req, db, ref, logging}) => {
  // WRITE YOUR CODE ONLY BELOW THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
  logging.log("basicParser started")
  try {
  // Import any NPM package needed
  const lodash = require('lodash');
  logging.log(lodash.join(['a', 'b', 'c'], '~'))
  } catch (e){
    logging.error(e);
  }
  
  // WRITE YOUR CODE ONLY ABOVE THIS LINE. DO NOT WRITE CODE/COMMENTS OUTSIDE THE FUNCTION BODY
}
```

**Reply by hrutik7** [reply, marked as answer]
hey @ralphilius 
the syntax should be await import
for reference :
https://github.com/rowyio/rowy/commit/c1295926204e9bb73d17d310885c510f76224501
